lightning-background-processor/src/lib.rs

```diff
@@ -91,20 +97,14 @@ where
 	///
 	/// [`ChannelManager`]: lightning::ln::channelmanager::ChannelManager
 	fn persist_manager(&self, channel_manager: &ChannelManager<Signer, M, T, K, F, L>) -> Result<(), std::io::Error>;
-}
 
-impl<Fun, Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
-ChannelManagerPersister<Signer, M, T, K, F, L> for Fun where
-	M::Target: 'static + chain::Watch<Signer>,
-	T::Target: 'static + BroadcasterInterface,
-	K::Target: 'static + KeysInterface<Signer = Signer>,
-	F::Target: 'static + FeeEstimator,
-	L::Target: 'static + Logger,
-	Fun: Fn(&ChannelManager<Signer, M, T, K, F, L>) -> Result<(), std::io::Error>,
-{
-	fn persist_manager(&self, channel_manager: &ChannelManager<Signer, M, T, K, F, L>) -> Result<(), std::io::Error> {
-		self(channel_manager)
-	}
+	/// Persist the given [`NetworkGraph`] to disk, logging an error if persistence failed.
+	/// Unlike, [`persist_manager`], this will not cause [`BackgroundProcessor`] to exit.
+	/// 
+	/// [`NetworkGraph`]: lightning::routing::network_graph::NetworkGraph
+	/// [`BackgroundProcessor`]: lightning-background-processor::BackgroundProcessor
```

Comment:
You should be able to drop this, no? By default the docs stuff will resolve any symbols which are resolvable locally.

Reply:
got it -- this is my first time working with `rustdoc` works so you're probably right 😄